aea/cli/remove.py

```diff
@@ -38,16 +38,23 @@ def remove(ctx: Context):
 
 def _remove_item(ctx: Context, item_type, item_name):
     """Remove an item from the configuration file and agent."""
+    # try to parse the item_name as public id
+    try:
+        item_id = PublicId.from_string(item_name)
+        item_name = item_id.name
+    except ValueError:
+        item_id = item_name
```

Comment:
drop this line and in 44 add item_name = item_id.name so we can call it what it is?